app-creativity/librecad/autobuild/defines

```diff
@@ -0,0 +1,5 @@
+PKGNAME=librecad
+PKGDEP="gcc-runtime qt-5"
+BUILDDEP="boost"
+PKGSEC="graphics"
+PKGDES="2D CAD program written in C++17"
```

Comment:
There is no point advertising to the user that this program was written in C++17.